File: docs/components_page/components/badge/color.jl
using DashBootstrapComponents, DashHtmlComponents

badges = html_span([
    dbc_badge("Primary", color = "primary", className = "me-1"),
    dbc_badge("Secondary", color = "secondary", className = "me-1"),
    dbc_badge("Success", color = "success", className = "me-1"),
    dbc_badge("Warning", color = "warning", className = "me-1"),
    dbc_badge("Danger", color = "danger", className = "me-1"),
    dbc_badge("Info", color = "info", className = "me-1"),
    dbc_badge("Light", color = "light", text_color = "dark", className = "me-1"),
    dbc_badge("Dark", color = "dark"),
]);

File: docs/components_page/components/badge/color.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

badges <- htmlSpan(
  list(
    dbcBadge("Primary", color = "primary", className = "me-1"),
    dbcBadge("Secondary", color = "secondary", className = "me-1"),
    dbcBadge("Success", color = "success", className = "me-1"),
    dbcBadge("Warning", color = "warning", className = "me-1"),
    dbcBadge("Danger", color = "danger", className = "me-1"),
    dbcBadge("Info", color = "info", className = "me-1"),
    dbcBadge("Light", text_color = "dark", color = "light", className = "me-1"),
    dbcBadge("Dark", color = "dark")
  )
)
